Automattic/woocommerce-payments
# Include deposit status in Transaction exports

Fixes #2715

#### Changes proposed in this Pull Request
As mentioned in #2715, this adds a deposit_status column to Transaction exports. Additionally, this change replaces `client/deposits/strings.js` with a typescript file.
There's a separate server PR to expose deposit_status in the transaction response at 1385-gh-Automattic/woocommerce-payments-server








#### Testing instructions





* As an admin user with deposit records already available in Payments -> Deposits
* Go to Payments -> Transactions
* Click Download and open the exported csv file
* Notice the new Deposit Status column
* Ensure that deposit status is correct for the relevant transaction

-------------------


**Post merge**
